libnmstate/nm/device.py

```diff
@@ -20,13 +20,28 @@
 from . import nmclient
 
 
-def activate(dev, connection=None):
+def activate(dev, fetch_remote_connection_func=None):
```

Comment:
Shouldn't the dev parameter be optional?

Reply:
It can be. I'll change it.